include/spirv-tools/libspirv.h

```diff
@@ -73,6 +73,7 @@ typedef enum spv_result_t {
   SPV_ERROR_INVALID_CAPABILITY = -13,
   SPV_ERROR_INVALID_DATA = -14,  // Indicates data rules validation failure.
   SPV_ERROR_MISSING_EXTENSION = -15,
+  SPV_ERROR_WRONG_VERSION = -16, // Indicates wrong SPIR-V version
```

Comment:
I'm not sure about whether to introduce this new error type. It just seems there is no suitable existing ones.